[1.0] docs: change an ambiguous part of "application order" section (#354)
vrm-c/vrm-specification: specification/VRMC_vrm-1.0-beta/README.md

[./firstPerson.md](./firstPerson.md)

## Expression, LookAt, SpringBone, Constraints execution order

* VRMC_vrm.lookAt
* VRMC_vrm.expression
* VRMC_node_constraint
* VRMC_springBone

> Has changed to Node and Mesh, which affects the execution order.
> The recommended order of application of updates is as follows.

Author: ついでに、ちょっと英文おかしそうだったのでこちらも修正します。

```suggestion
These components affect Node and Mesh, and the execution order matters.
The recommended order of execution in an update loop is as follows.
```

> ## Expression, LookAt, SpringBone, Constraints application order

Author: See: https://github.com/vrm-c/vrm-specification/pull/354#issuecomment-1050388775

```suggestion
## Expression, LookAt, SpringBone, Constraints execution order
```
